tests/justfile

```diff
@@ -4,6 +4,11 @@ test:
 
 alias tests := test
 
+test-slinky:
+    # NOTE: if you change the -t flag value, you MUST also change the name in slinky_test.go L29.
+    cd .. && go mod tidy && docker build -f warden.Dockerfile -t warden/warden-protocol .
+    cd slinky && go mod tidy && go test -v -race .
+
```

Comment:
this runs the integration suite. please note the comment. the test suite relies on the tag of the docker file to be `warden/warden-protocol`. if this is not desirable, the test will need to be updated as well.